spec/index.bs

```diff
@@ -2428,114 +2428,115 @@ A {{GPUBindGroupLayout}} object has the following internal slots:
-            are undefined.
-        1. If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/type}} is **not**
-            {{GPUBindingType/"readonly-storage-texture"}}
-            or {{GPUBindingType/"writeonly-storage-texture"}},
-            ensure
-            |bindingDescriptor|.{{GPUBindGroupLayoutEntry/storageTextureFormat}}
-            is `undefined`.
-        1. If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/type}} is {{GPUBindingType/uniform-buffer}}:
-            1. Ensure [=uniform buffer validation=] is not violated.
-            1. If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/hasDynamicOffset}} is `true`, ensure [=dynamic uniform buffer validation=] is not violated.
-        1. If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/type}} is {{GPUBindingType/storage-buffer}} or {{GPUBindingType/readonly-storage-buffer}}:
-            1. Ensure [=storage buffer validation=] is not violated.
-            1. If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/hasDynamicOffset}} is `true`, ensure [=dynamic storage buffer validation=] is not violated.
-        1. If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/type}} is {{GPUBindingType/sampled-texture}}
-            1. Ensure [=sampled texture validation=] is not violated.
-            1. If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/multisampled}} is `true`, ensure [=multisample texture view dimension validation=] is not violated.
-        1. If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/type}} is {{GPUBindingType/readonly-storage-texture}} or {{GPUBindingType/writeonly-storage-texture}}:
-            1. Ensure [=storage texture validation=] is not violated.
-            1. Ensure [=storage texture format validation=] is not violated.
-        1. If |bindingDescriptor|.{{GPUBindGroupLayoutEntry/type}} is {{GPUBindingType/sampler}} or {{GPUBindingType/comparison-sampler}}
-            , ensure [=sampler validation=] is not violated.
-        1. Insert |bindingDescriptor| into |layout|.{{GPUBindGroupLayout/[[entryMap]]}}
-            with the key of |bindingDescriptor|/{{GPUBindGroupLayoutEntry/binding}}.
-    1. Return |layout|.
-
-    <div class=validusage dfn-for=GPUDevice.createBindGroupLayout>
-        <dfn abstract-op>Valid Usage</dfn>
-
-        If any of the following conditions are violated:
-            1. Generate a {{GPUValidationError}} in the current scope with appropriate error message.
-            1. Create a new [=invalid=] {{GPUBindGroupLayout}} and return the result.
-
-        <dfn>bind group layout device validation</dfn>: The {{GPUDevice}} must not be lost.
-
-        <dfn>binding validation</dfn>: Each |bindingDescriptor|.{{GPUBindGroupLayoutEntry/binding}} in |descriptor| must be unique.
-
-        <dfn>vertex shader binding validation</dfn>: {{GPUBindingType/storage-buffer}} and {{GPUBindingType/writeonly-storage-texture}} are not allowed.
+<dl dfn-type=method dfn-for=GPUDevice>
+    : <dfn>createBindGroupLayout(descriptor)</dfn>
```

Comment:
I have no idea why, but in the PR preview bikeshed seems to have created another nested `<dl></dl>` surrounding just this line.

![image](https://user-images.githubusercontent.com/606355/89349915-e2005380-d663-11ea-9648-40d4015b4ef3.png)

Reply:
This was due to the method description and algorithm not being indented, apparently.